src/Game.cpp

```diff
@@ -27,24 +28,62 @@ void Game::setIsStrikeOrSpare(unsigned short int position, const std::string& va
     }
 }
 
-int Game::checkValue(const std::string & value)
+int Game::checkValue(const std::string& value)
 {
-    int valueInt;
-    auto valueTemp=value.substr(0,2);
-    if ((value[0] == 'X') || (value[1] == '/'))
+    int firstBall, secondBall;
+    std::tie(firstBall, secondBall) = convertValueToInt(value);
+    return firstBall + secondBall;
+}
+
+std::pair <int, int> Game::convertValueToInt(const std::string& value)
```

Comment:
The function name does not tell what this function is doing. 
convertValueToInts or convertValueToPairOfInts is more proper now.